libminifi/src/core/logging/Logger.cpp

```diff
@@ -100,7 +100,7 @@ bool Logger::should_log(const LOG_LEVEL &level) {
 void Logger::log_string(LOG_LEVEL level, std::string str) {
   switch (level) {
     case critical:
-      log_warn(str.c_str());
+      log_critical(str.c_str());
```

Comment:
wow, good catch -- good thing we don't use `log_critical` much (or at all?)

Reply:
yeah, I dont think we use it anywhere but if in the future we encounter something that would warrant a critical log we sure as hell wouldnt want to miss it by accident